src/main/java/ch/njol/skript/structures/StructCommand.java

```diff
@@ -259,6 +260,7 @@ public boolean load() {
 		}
 
 		String description = entryContainer.get("description", String.class, true);
+		String prefix = entryContainer.getOptional("prefix", String.class, true);
```

Comment:
This should technically be false since there is no default value (this is mainly just for formatting)
```suggestion
		String prefix = entryContainer.getOptional("prefix", String.class, false);
```